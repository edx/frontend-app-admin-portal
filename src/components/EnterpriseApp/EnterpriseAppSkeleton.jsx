import React from 'react';
import Skeleton from 'react-loading-skeleton';

const EnterpriseAppSkeleton = () => (
  <>
    <Skeleton height="8rem" />
    <Skeleton height="70vh" />
  </>
);

export default EnterpriseAppSkeleton;
