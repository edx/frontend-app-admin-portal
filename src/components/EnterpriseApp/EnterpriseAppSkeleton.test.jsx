import React from 'react';
import renderer from 'react-test-renderer';
import EnterpriseAppSkeleton from './EnterpriseAppSkeleton';

describe('EnterpriseAppSkeleton', () => {
  it('renders a skeleton', () => {
    const tree = renderer
      .create((
        <EnterpriseAppSkeleton />
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
