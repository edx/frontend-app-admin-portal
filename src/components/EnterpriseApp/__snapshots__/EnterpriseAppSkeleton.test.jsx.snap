// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnterpriseAppSkeleton renders a skeleton 1`] = `
Array [
  <div
    className="sr-only"
  >
    Loading...
  </div>,
  <span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": "8rem",
        }
      }
    >
      ‌
    </span>
  </span>,
  <span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": "70vh",
        }
      }
    >
      ‌
    </span>
  </span>,
]
`;
